healthcare/v1/src/test/java/snippets/healthcare/DatasetTests.java

```diff
@@ -63,6 +65,22 @@ public static void checkRequirements() {
     requireEnvVar("GOOGLE_CLOUD_PROJECT");
   }
 
+  @AfterClass
+  public static void deleteTempItems() throws IOException {
+    // Delete the destination dataset created during de-identification.
+    try {
+      DatasetDelete.datasetDelete(destinationDatasetName);
+    } catch (GoogleJsonResponseException ex) {
+      if (ex.getStatusCode() == 404) {
+        System.out.println("Destination dataset already deleted, continue.");
+      } else {
+        System.out.println(
```

Comment:
If it was some other error,  we must surface that error. So I think we should throw the exception in the else case. My suggestion is to have something as below to have the tests clean and simple.

```java
} catch (GoogleJsonResponseException ex) {
     // if 404, then the dataset has already been deleted
      if (ex.getStatusCode() != 404) {
           throw ex;
      }
}
```